packages/@okta/vuepress-site/docs/guides/authenticators-custom-authenticator/main/index.md

```diff
@@ -64,13 +69,13 @@ The following image shows the Devices SDK setup in the Admin Console:
 
 ### Create an OIDC web authentication client
 
-The simplest way to integrate authentication in your app is with OIDC through a web browser, using the [Authorization Code flow grant type](/docs/guides/implement-grant-type/authcode/main/). You need an access token to start the enrollment flow for the Devices SDK. For future sign-in attempts, consider using refresh tokens. <StackSnippet snippet="samplecode" inline />
+The simplest way to integrate authentication in your app is with OIDC through a web browser, using the [Authorization code flow grant type](/docs/guides/implement-grant-type/authcode/main/). You need an access token to start the enrollment flow for the Devices SDK. For future sign-in attempts, consider using refresh tokens. <StackSnippet snippet="samplecode" inline />
 
-### Grant the required scopes
+<StackSnippet snippet="jwtbearernote" />
 
-> **Note:** You must use an [Org Authorization Server](/docs/concepts/auth-servers/#available-authorization-server-types) to grant the scopes needed to create a custom authenticator. Custom Authorization Servers (including the default Custom Authorization Server) don't work.
+### Grant the required scopes
```

Comment:
@danielmaharry-okta In this section, we need to modify the scopes to use the new `okta.myAccount.appAuthenticator.manage, okta.myAccount.appAuthenticator.read` scopes. Will that be included in this PR?

Reply:
@jane-kuang-okta I've added it to the docs.